### DISCUSSION TITLE
Why can't I dismiss my own reviews?

### DISCUSSION THREAD

**Discussion by tjprescott** [community/community#120006] [Pull Requests]
### Select Topic Area

Product Feedback

### Body

On occasion, I come across a PR that is approved and I notice a red flag and throw a "Request Changes" review on it to prevent it being merged. In those cases, I simply want the original reviewers to take a second look at the issue I raised and invite them to dismiss my review. I was surprised today to find that I could not dismiss my own "Request Changes" review.  GH Copilot suggested it was becuase I don't have the necessary permissions, but it seems odd that I would need any permission to dismiss my own review.

I specifically didn't want to "Approve" the PR since I didn't want to create a false impression that I had fully reviewed it or endorsed the changes, so I tried to replace the review with a "Comment" review, but that also did not work.

It seems to me that if you have permission to create a review then you should have the permission to dismiss it as well.

**Comment by tklever** [comment]
I just ran into this issue, nice to see the feature request is already here. 

As far as my use case, it comes from the inevitability of change. In large organizations, over time developers move teams, move projects, change roles, and responsibilities. A reviewer's access, permissions, and team membership could change over the lifecycle of a PR. A reviewer might no longer be the appropriate reviewer, and requested changes might no longer be vaild.

Where there's friction is with these changes, a reviewer might fall out of the groups allowed to dismiss reviews in branch protection rules, so even though a reviewer might be trying to eliminate friction by stepping aside and into their new role, they're prevented from doing so until they find an appropriate party across all the repos where they have outstanding reviews (this might be many teams with many different permutations of branch protection rules)
 
This feature request would override the dismissal settings in branch protection rules and always all a reviewer to dismiss their own review. Branch protection rules still apply, but a reviewer can always dismiss their own work.

**Comment by thomagos** [comment]
Someone please fix this. GitKraken is full of old, dead repositories with their dead PRs I reviewed some weeks, months and years ago - yet I can't remove myself to cleanup the mess.

**Comment by jeremy-spiller-tfs** [comment]
I would like to be able to dismiss my "request changes" review without having to approve the PR.
